cloud/blockstore/libs/nbd/netlink_device.cpp

```diff
@@ -183,8 +183,19 @@ class TNetlinkDevice final
     {
         Log = Logging->CreateLog("BLOCKSTORE_NBD");
 
-        if (sscanf(DeviceName.c_str(), "/dev/nbd%u", &DeviceIndex) != 1) {
-            throw TServiceError(E_ARGUMENT) << "invalid nbd device target";
+        const char suffix[] = "nbd";
+        size_t index = DeviceName.find(suffix);
```

Comment:
const
но вообще кажется без переменной можно обойтись

Reply:
А как, она же там дальше используется? Да и зачем?

Reply:
У TString[Buf] есть метод ChopSuffix:
```cpp
TStringBuf path{DeviceName};

if (!path.ChopSuffix("nbd")) { ... }

TStringStream stream(path);
```

Reply:
ChopSuffix не то делает
```
    // returns true if string ended with `suffix`, false otherwise
    inline bool ChopSuffix(const TdSelf& suffix) noexcept {
        return BeforeSuffix(suffix, *this);
    }
```